api/capi/src/nnstreamer-capi-single.c

```diff
@@ -401,7 +411,6 @@ ml_single_open (ml_single_h * single, const char *model,
   }
 
   g_mutex_lock (&single_h->lock);
```

Comment:
plz remove these two lines also (nothing to do here)

Reply:
done.